src/plugins/intel_gna/src/serial/gna_model_serial.cpp

```diff
@@ -563,61 +609,49 @@ void GNAModelSerial::Export(const GnaAllocations& allocations, std::ostream& os)
         header_latest::RuntimeEndPoint ep = ReadEndPoint(is);
 
-        input.ptrs.push_back(reinterpret_cast<float*>(reinterpret_cast<uint8_t*>(basePtr) + ep.descriptor_offset));
-        input.orientation = ep.orientation;
-        input.num_elements = ep.elements_count;
-        input.scale_factor = ep.scaleFactor;
-        input.model_precision =
+        node.ptrs.push_back(reinterpret_cast<float*>(reinterpret_cast<uint8_t*>(base_ptr) + ep.descriptor_offset));
+        node.orientation = ep.orientation;
+        node.num_elements = ep.elements_count;
+        node.scale_factor = ep.scaleFactor;
+        node.model_precision =
             InferenceEngine::Precision(static_cast<InferenceEngine::Precision::ePrecision>(ep.precision));
-        input.set_precision(ep.element_size);
-        input.model_layout = static_cast<InferenceEngine::Layout>(ep.layout);
-        input.allocated_size = input.get_required_size();
+        node.set_precision(ep.element_size);
+        node.model_layout = static_cast<InferenceEngine::Layout>(ep.layout);
+        node.allocated_size = node.get_required_size();
 
         auto inputDims = InferenceEngine::SizeVector();
         for (auto i = 0; i < ep.shape.NumberOfDimensions; ++i) {
             inputDims.push_back(ep.shape.Dimensions[i]);
         }
-        input.dims = inputDims;
+        node.dims = inputDims;
 
         // read tensor names
         for (uint8_t tId = 0; tId < ep.tensor_names_count; ++tId) {
-            input.tensor_names.insert(readString(is));
-        }
-
-        AppendTensorNameIfNeeded(input);
-    }
-}
-
-void GNAModelSerial::ImportOutputs(std::istream& is, void* basePtr, GnaOutputs& outputs) {
-    for (auto& output : outputs.Get()) {
-        header_latest::RuntimeEndPoint ep = ReadEndPoint(is);
-
-        output.ptrs.push_back(reinterpret_cast<float*>(reinterpret_cast<uint8_t*>(basePtr) + ep.descriptor_offset));
-        output.orientation = ep.orientation;
-        output.num_elements = ep.elements_count;
-        output.scale_factor = ep.scaleFactor;
-        output.set_precision(ep.element_size);
-        output.model_precision =
-            InferenceEngine::Precision(static_cast<InferenceEngine::Precision::ePrecision>(ep.precision));
-        output.model_layout = static_cast<InferenceEngine::Layout>(ep.layout);
-        output.allocated_size = output.get_required_size();
-
-        auto outputDims = InferenceEngine::SizeVector();
-        for (auto i = 0; i < ep.shape.NumberOfDimensions; ++i) {
-            outputDims.push_back(ep.shape.Dimensions[i]);
+            node.tensor_names.insert(readString(is));
         }
-        output.dims = outputDims;
-
-        // read tensor names
-        for (uint8_t tId = 0; tId < ep.tensor_names_count; ++tId) {
-            output.tensor_names.insert(readString(is));
+        AppendTensorNameIfNeeded(node);
+
+        // read preprocessing model
+        if (model_header_.version.major == 2 && model_header_.version.minor >= 9) {
```

Comment:
what about using ov::intel_gna::header_latest::IsFirstVersionLower 
e.g.
```
    static constexpr header_2_dot_9::ModelHeader::Version preprocesing_allowed_version;
    if (!header_latest::IsFirstVersionLower(model_header_.version, preprocesing_allowed_version)) {
    }
```

Reply:
I don't see any benefits to use this help function, code would be less readable, those versions will not be changed in future.